src/Pelagos/Bundle/AppBundle/Resources/public/js/gMap.js

```diff
@@ -18,39 +18,60 @@
             });
         });
     };
-    
+
     function renderMap(map, wkt) {
         var raster = new ol.layer.Tile({
             source: new ol.source.OSM()
         });
-        
+
         var format = new ol.format.WKT();
-        
+
         var googleLayerSatellite = new ol.layer.Tile({
             title: "Google Satellite",
-            source: new ol.source.TileImage({ 
-                url: "https://mt1.google.com/vt/lyrs=s&hl=pl&&x={x}&y={y}&z={z}" 
+            source: new ol.source.TileImage({
+                url: "https://mt1.google.com/vt/lyrs=y&hl=en&&x={x}&y={y}&z={z}"
             }),
         });
 
         var feature = format.readFeature(wkt, {
             dataProjection: "EPSG:4326",
             featureProjection: "EPSG:3857"
         });
-            
+
+        var style = new ol.style.Style({
+            fill: new ol.style.Fill({
+              color: 'rgba(255, 255, 255, 0)'
```

Comment:
eslint complains about single quotes used in this section, otherwise looks good.